packages/react-core/src/components/Tooltip/examples/TooltipOptions.tsx

```diff
@@ -0,0 +1,205 @@
+              checked && (updatedTrigger = trigger.concat('mouseenter'));
+              !checked && (updatedTrigger = trigger.filter(t => t !== 'mouseenter'));
+              setIsVisible(false);
+              setTrigger(updatedTrigger);
+            }}
+            aria-label="trigger: mouseenter"
+            id="trigger_mouseenter"
+          />
+          <Checkbox
+            label="trigger: focus"
+            isChecked={trigger.includes('focus')}
+            onChange={checked => {
+              let updatedTrigger;
+              checked && (updatedTrigger = trigger.concat('focus'));
+              !checked && (updatedTrigger = trigger.filter(t => t !== 'focus'));
+              setIsVisible(false);
+              setTrigger(updatedTrigger);
+            }}
+            aria-label="trigger: focus"
+            id="trigger_focus"
+          />
+          <Checkbox
+            label="trigger: click"
+            isChecked={trigger.includes('click')}
+            onChange={checked => {
+              let updatedTrigger;
+              checked && (updatedTrigger = trigger.concat('click'));
+              !checked && (updatedTrigger = trigger.filter(t => t !== 'click'));
+              setIsVisible(false);
+              setTimeout(() => setTrigger(updatedTrigger));
+            }}
+            aria-label="trigger: click"
+            id="trigger_click"
+          />
+          <Checkbox
+            label="trigger: manual"
+            isChecked={trigger.includes('manual')}
+            onChange={checked => {
+              let updatedTrigger;
+              checked && (updatedTrigger = trigger.concat('manual'));
+              !checked && (updatedTrigger = trigger.filter(t => t !== 'manual'));
+              setIsVisible(false);
+              setTrigger(updatedTrigger);
+            }}
+            aria-label="trigger: manual"
+            id="trigger_manual"
+          />
+        </div>
+        <div style={{ border: '1px solid' }}>
+          <Checkbox
+            label="content left-aligned"
+            isChecked={contentLeftAligned}
+            onChange={checked => setContentLeftAligned(checked)}
+            aria-label="content left-aligned"
+            id="content_left_aligned"
+          />
+        </div>
+        <div style={{ border: '1px solid' }}>
+          <Checkbox
+            label="enableFlip"
+            isChecked={enableFlip}
+            onChange={checked => setEnableFlip(checked)}
+            aria-label="enableFlip"
+            id="enable_flip"
+          />
+        </div>
+        <div style={{ border: '1px solid' }}>
+          position (will flip if enableFlip is true). The 'auto' position requires enableFlip to be set to true.
+          <Select
+            onToggle={() => setPositionSelectOpen(!positionSelectOpen)}
+            onSelect={(_event, selection) => {
+              setPosition(selection);
+              setPositionSelectOpen(false);
+            }}
```

Comment:
I think if you update the following lines it will help:
```
onSelect={(_event: React.MouseEvent | React.ChangeEvent, selection: string | SelectOptionObject) => {
  setPosition(selection.toString() as TooltipPosition);
  setPositionSelectOpen(false);
}}
```